Trivial fix to prevent int overflow (required to build with -werror).

Signed-off-by: Chris Smith <chris.smith@st.com>

Index: gdb-6.8/gdb/value.c
===================================================================
--- gdb-6.8.orig/gdb/value.c	2008-09-19 12:14:57.000000000 +0100
+++ gdb-6.8/gdb/value.c	2008-09-19 12:17:04.000000000 +0100
@@ -661,7 +661,7 @@
 {
   int i;
   struct value *val;
-  static int num = 1;
+  static unsigned int num = 1;
 
   if (num_exp)
     {
Index: gdb-6.8/gdb/memattr.c
===================================================================
--- gdb-6.8.orig/gdb/memattr.c	2008-09-11 10:35:35.000000000 +0100
+++ gdb-6.8/gdb/memattr.c	2008-09-19 12:41:40.000000000 +0100
@@ -174,7 +174,7 @@
 		   const struct mem_attrib *attrib)
 {
   struct mem_region new;
-  int i, ix;
+  unsigned int i, ix;
 
   /* lo == hi is a useless empty region */
   if (lo >= hi && hi != 0)
